### PR TITLE
update org readme

### PR DESCRIPTION
**Pull request by tuves**
main page readme updates

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
 ![image](https://user-images.githubusercontent.com/6633808/160689302-3fe5e5d4-ba24-4525-8ed1-a8351ccbc0ef.png)
 
-# GitHub Community
+<p align="center">
+  <img width="612" height="135" src="https://user-images.githubusercontent.com/49915749/179235266-19730a30-47bf-47b1-9456-e03f64cfc0dd.png">
```
Comment (jenna-wolf) [inline profile/README.md:4]: @tuves we will not be able to include this lockup here as it is to only be used for promotional purposes. So we can just keep the headline without the logo.

**Review by jenna-wolf — CHANGES_REQUESTED** [review]
@tuves  I would just remove the lockup and then remember to include the category post link. Then we should be set! Thanks so much :heart:

**Review by ettaboyle — APPROVED** [review]
Approved 🚀

**Comment by IsaqueDato** [comment]
Gente eu tô bem perdido, nunca soube pra onde foi programação, mas é minha chance não posso perder.

**Pull request merged**